spawn_sdk/spawn_sdk/README.md

```diff
@@ -82,17 +82,19 @@ defmodule SpawnSdkExample.Actors.MyActor do
 
   alias Io.Eigr.Spawn.Example.{MyState, MyBusinessMessage}
 
-  defact init(%Context{state: state} = ctx) do
+  # The callback could also be referenced to an existing function:
+  # action "init", &some_defp_handler/0
+  # action "init", &SomeModule.handler/1
+  # action "init", &SomeModule.handler/2
+
+  action "init", fn %Context{state: state} = ctx ->
     Logger.info("[joe] Received InitRequest. Context: #{inspect(ctx)}")
 
     Value.of()
     |> Value.state(state)
   end
 
-  defact sum(
-           %MyBusinessMessage{value: value} = data,
-           %Context{state: state} = ctx
-         ) do
+  action "sum", fn %Context{state: state} = ctx, %MyBusinessMessage{value: value} = data ->
```

Comment:
How do I guarantee the order of the parameters? Is this forced somehow by the SDK or is it based on convention?